setup.py

```diff
@@ -37,20 +37,22 @@
 
 install_requires = [
     'inspire-json-merger~=11.0,>=11.0.0',
-    'dictdiffer>=0.9.0',
+    'dictdiffer<0.9.0; python_version <= "2.7"',
+    'dictdiffer>=0.9.0; python_version >= "3.6"',
     'inspire-utils~=3.0,>=3.0.0',
-    'invenio-search>=1.0.0a10',
+    'invenio-search>=1.2.3',
```

Comment:
I'm really afraid about that

Reply:
We use this or newer versions in next and hep, so it should🤞be fine:

* https://github.com/inspirehep/inspire-next/blob/master/pyproject.toml#L69
* https://github.com/inspirehep/inspirehep/blob/master/backend/pyproject.toml#L47